app/controllers/topics_controller.rb

```diff
@@ -1,7 +1,56 @@
+require 'date'
+require 'json'
+require 'typhoeus'
```

Comment:
All of this should be required by bundler already. Please remove and try it out. It should work just as fine